src/yashCJun20/Assignment19/FindIndex.java

```diff
@@ -0,0 +1,24 @@
+package yashCJun20.Assignment19;
+
+public class FindIndex {
+	
+	void toFindDifferIndex (int [] arr1,int [] arr2) {
+		
+		for (int i=0;i<arr1.length;i++) {
+		
+        	if (arr1[i]!=arr2[i]) 
```

Comment:
What is the difference between DifferIndex class and FindIndex class.

Reply:
Give Proper method name.

Reply:
As advised,changes made to the code.
Please review.
I will take care of this for the upcoming assignments